Misc/NEWS.d/next/Library/2024-09-30-04-49-22.gh-issue-124529.hDDlMH.rst

```diff
@@ -0,0 +1,9 @@
+Fix :meth:`datetime.datetime.strptime`, :meth:`datetime.date.strptime` as
+well as :func:`time.strptime` (by modifying :class:`_strptime.TimeRE`) to
```

Comment:
Maybe you shouldn't write classes or methods with underscores to NEWS, because they're not public.
And, Maybe you can try to shorten the description of NEWS, it's too long.

Reply:
OK, shortened and the reference to a non-public API removed. :)